doc/devel/architecture.md

```diff
@@ -67,30 +42,6 @@ device information to create the definitive Device object that will be stored in
 Particular IoT Agents _may_ support autoregistration of devices into configurations, if enough information is given from
 the Southbound.
 
-#### Configurations and subservices
-
-Configurations are meant to be a mean of simplifying the device provisioning for groups of very similar devices.
-Considering that different groups of devices may be created in the same subservice that may require different
-configurations, multiple configurations are allowed for each subservice. Considering the key in the association between
-Device and Configuration was the triplet (service, subservice, type), all of these elements are considered mandatory.
-
-This statement doesn't hold true for older IoT Agents, though. In older versions of the IoT Agents, each device
-configuration was assigned to a particular subservice and just one configuration was allowed per subservice, so the
-relation between a Device and a Configuration didn't need the type to discriminate between Configurations. That's why
-for those agents, type was not a mandatory parameter.
-
-In order to allow backward-compatibility with those agents, the IoT Agent Library now implement a compatibility mode:
-the **Single Configuration Mode**, that makes the agent behave like the old agents. In this mode:
-
--   Each Subservice can contain just one Configuration. If a second Configuration is created for a Subservice, an error
-    is raised.
-
--   Each Device provisioned for a Subservice is automatically assigned to the Subservice one Configuration if there is
-    any.
-
-This compatibility has to be set for the whole IoT Agent, and there is no option of having both modes simultaneously
-running. Transitions from one mode to the other should be made with care, and may involve data migration.
-
```

Comment:
Deleted. It does not make sense
https://github.com/telefonicaid/iotagent-node-lib/issues/1469